api/docs/building.dox

```diff
@@ -47,7 +47,7 @@ To build DynamoRIO on Linux, use the following commands as a guide.  This builds
   # other distributions (in particular, use "cmake3" for Ubuntu Trusty).
   $ sudo apt-get install cmake g++ g++-multilib doxygen git zlib1g-dev libunwind-dev libsnappy-dev liblz4-dev
   # Get sources.
-  $ git clone https://github.com/DynamoRIO/dynamorio.git
+  $ git clone --recurse https://github.com/DynamoRIO/dynamorio.git
```

Comment:
nit: I think the actual option is either --recursive or --recurse-submodules?  I guess it is allowing your abbreviation but maybe better to use the real name.

Reply:
Done.